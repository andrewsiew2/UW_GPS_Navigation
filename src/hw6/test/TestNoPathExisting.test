# Tries to find a path from a node to another node whose path does not exist

LoadGraph tester testing.tsv

FindPath tester LUTHER SUPERMAN