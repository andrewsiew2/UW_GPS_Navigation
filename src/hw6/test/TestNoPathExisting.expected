# Tries to find a path from a node to another node whose path does not exist

loaded graph tester

path from LUTHER to SUPERMAN:
no path found